lib/fhir_client.rb

```diff
@@ -1,4 +1,6 @@
 require 'fhir_models'
+require 'fhir_models/r4b'
```

Comment:
I'm sorry, Diego, I thought I removed this comment before I finished my review because I decided it was out of scope. Could you undo this part of the changes?